### DISCUSSION TITLE
Sugestão Curso 2806

### DISCUSSION THREAD

**Discussion by rafamalta** [balta-io/forum#298] [Tira Dúvidas] [answered]
Bom dia, Professor. Me deparei com uma situação e gostaria de deixar uma sugestão referente ao Curso 2806 - Acesso à dados com C#, .NET 5, Dapper e SQL Server, do seguinte link: https://github.com/balta-io/2806/tree/main/Desafio

Ocorre que o código fonte do Desafio Final não está completo aqui no GitHub. Seria interessante completá-lo para que possamos ver como realizar cada uma das funcionalidades propostas.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rafamalta, como vai?

Muito obrigado pela sugestão. Na minha opinião, acredito que não há necessidade de completar tudo, é apenas mais do mesmo.
Mas se tiver alguma implementação que queira me mostrar, pode mandar aqui que valido com certeza 🚀
